CS-832: Update workflows

rvdh opened the pull request.

ChrisTerBeke commented on .github/workflows/publish.yaml:
```diff
@@ -0,0 +1,34 @@
+    if: "!contains(github.ref, 'beta')"
+    runs-on: ubuntu-latest
+    env:
+      CLOUDSMITH_TOKEN: ${{ secrets.CLOUDSMITH_TOKEN }}
```
I think we publish our non-private packages to NPM instead of CloudSmith.

ChrisTerBeke replied:
We could change that however? Not sure what the other @Ultimaker/stardust_frontend peeps think.

rvdh replied:
Ah right, for now I'll add a distinction between the NPM and CloudSmith packages.

daansmit replied:
I do not know enough about the public packages that Ultimaker has to judge this to be honest.

rvdh commented:
@ChrisTerBeke @daansmit I've added an empty 'build' script to the package.json so we can reuse the same workflow file for all projects. I also added a .npmrc which configures where the package will be published to.
